### PR TITLE
Opening up security - Vault post

### PR DESCRIPTION
**Pull request by flock3**
Vault article, detailing a short overview of how it was implemented recently in a client, some of the problems the team came across and how they eventually solved them.

Requesting any feedback on content or grammar, happy to modify before merge.

### CODE REVIEW

File: _posts/2017-02-14-opening-up-security.md [hunk 1]
```diff
@@ -0,0 +1,106 @@
...
+The team chose to use Vault by Hashicorp initially just to store the key/value pair secrets that they needed. Vault solved the problem of 
+how to store the secrets securely, and even partly solved the problem of how to get them onto the containers; but there was still some not small
+work to truly integrate Vault into their workflow.
+
+Now when the team is developing the software and they have a new secret to store, there is very little temptation just to store it in the source
+or any other easy solution, because storing it in the Vault that lives in the environment, is not that much more complicated.
+
+## How they authenticate and authorize
+
+One of the big problems with any system that manages any aspect of security, is authentication and authorization. Vault has a number of paradidms
```
Comment (florx) [inline _posts/2017-02-14-opening-up-security.md]: SP. paradigms?

Comment (flock3) [inline _posts/2017-02-14-opening-up-security.md]: Thanks @florx - good spot.

**Comment by Rucknar** [comment]
@flock3 Are you wanting more reviews on this or awaiting a merge?

**Comment by flock3** [comment]
Sorry no, I need to make some final updates based on review feedback, I will try and get these updates done in the next day

**Comment by flock3** [comment]
@Rucknar ready for merge

**Pull request merged**